lightning/src/ln/channel.rs

```diff
@@ -65,8 +65,6 @@ pub struct ChannelValueStat {
 }
 
 pub struct AvailableBalances {
-	/// The amount that would go to us if we close the channel, ignoring any on-chain fees.
-	pub balance_msat: u64,
```

Comment:
If we remove this, is there still a way to calculate our balance with msat precision since `ChannelMonitor::get_claimable_balances` will be rounded to sats?

Reply:
Not currently. With regard to `AvailableBalances::balance_msat`, its computation was very specific to a use case that is not applicable anymore and its value is likely not what is expected. For introspection at msat precision, perhaps `Channel::value_to_self_msat` and the pending HTLCs can be exposed. For the latter, see https://github.com/lightningdevkit/rust-lightning/pull/2442.

Reply:
> perhaps Channel::value_to_self_msat

Hm, perhaps. Will take a look at https://github.com/lightningdevkit/rust-lightning/pull/2442, thanks!

Reply:
I was thinking just to get our channel balance in general, not necessarily on-chain

Reply:
Ah okay. It's nice to have a specific use case for a balance IMO, pretty vague term in lightning. Maybe something like that would be useful for accounting reasons, though 🤷‍♀️

Reply:
Yea it's probably not a big deal to not have it for now